packages/block-library/src/gallery/edit.js

```diff
@@ -61,12 +61,28 @@ class GalleryEdit extends Component {
 		this.setImageAttributes = this.setImageAttributes.bind( this );
 		this.addFiles = this.addFiles.bind( this );
 		this.uploadFromFiles = this.uploadFromFiles.bind( this );
+		this.setAttributes = this.setAttributes.bind( this );
 
 		this.state = {
 			selectedImage: null,
 		};
 	}
 
+	setAttributes( attributes ) {
+		if ( attributes.ids ) {
+			throw new Error( "Don't use this attribute, dangit." );
```

Comment:
Note to self: even though this error should never be thrown, let's change the copy to something neutral and descriptive.